src/back/spv/image.rs

```diff
@@ -1017,7 +1017,7 @@ impl<'w> BlockContext<'w> {
                     Id::D3 => 3,
                 };
                 let extended_size_type_id = {
-                    let array_coords = if arrayed { 1 } else { 0 };
+                    let array_coords = usize::from(arrayed);
```

Comment:
thought: TIL about `impl From<bool> for {integer}`. 👀

Reply:
TIL also - I always just blasted to integer via `as`